### ISSUE TITLE
How to listener "Cancel Request".

### ISSUE THREAD

**Issue by kekedaine** [stephanenicolas/robospice#441]
How to listener "Cancel Request". please!

**Comment by johngamarra** [comment]
Hi, please read

https://github.com/stephanenicolas/robospice/wiki/Advanced-RoboSpice-Usages-and-FAQ#how-to-cancel-a-request-

**Comment by kekedaine** [comment]
@johngamarra : no, i want listen cancel action when i call spiceManager.cancel(myRequest);
